--- ChangeLog	2026-04-17 13:08:56.535096309 -0400
+++ ChangeLog-spell-corrected	2026-04-17 13:10:48.233242983 -0400
@@ -20,7 +20,7 @@
 
 	* .github/workflows/isort-and-black-checks.yml,
 	.github/workflows/macos.yaml, .github/workflows/ubuntu.yml,
-	.github/workflows/windows.yml, pyproject.toml: Project adminstration
+	.github/workflows/windows.yml, pyproject.toml: Project administration
 	stuff (#112) Repo is now Mathics3-scanner Use @v6 in CI. Bump min scanner version
 
 2026-03-13  rocky <rb@dustyfeet.com>
@@ -99,7 +99,7 @@
 2025-08-24  R. Bernstein <rocky@users.noreply.github.com>
 
 	* mathicsscript/__main__.py, mathicsscript/format.py,
-	mathicsscript/termshell.py, mathicsscript/termshell_prompt.py: 
+	mathicsscript/termshell.py, mathicsscript/termshell_prompt.py:
 	Miscellaneous small fixes... (#107) * Syntax Errors showing more properly by removing weird SymbolForm   handling. Also get ready for an "error" kind of format * Remove GNU Readline termshell style error when no style given * Better error checking on giving a file argument for an invalid
 	file
 
@@ -137,7 +137,7 @@
 2025-07-22  R. Bernstein <rocky@users.noreply.github.com>
 
 	* NEWS.md => CHANGES.rst, mathicsscript/bindkeys.py,
-	mathicsscript/termshell_prompt.py: Add f1 help (#101) * WIP - add f1 key * Addd f1 key and sync with release 9.0.0
+	mathicsscript/termshell_prompt.py: Add f1 key and sync with release 9.0.0
 
 2025-07-22  rocky <rb@dustyfeet.com>
 
@@ -338,7 +338,7 @@
 2023-02-25  rocky <rb@dustyfeet.com>
 
 	* mathicsscript/__main__.py, mathicsscript/asymptote.py,
-	mathicsscript/fake_psviewer.py, mathicsscript/format.py: Removce
+	mathicsscript/fake_psviewer.py, mathicsscript/format.py: Remove
 	flake8 things
 
 2023-02-25  rocky <rb@dustyfeet.com>
@@ -368,7 +368,7 @@
 
 2022-12-10  rocky <rb@dustyfeet.com>
 
-	* mathicsscript/termshell.py, mathicsscript/termshell_prompt.py: 
+	* mathicsscript/termshell.py, mathicsscript/termshell_prompt.py:
 	black
 
 2022-11-28  rocky <rb@dustyfeet.com>
@@ -474,7 +474,7 @@
 2022-02-15  rocky <rb@dustyfeet.com>
 
 	* NEWS.md, mathicsscript/bindkeys.py,
-	mathicsscript/termshell_gnu.py, mathicsscript/termshell_prompt.py: 
+	mathicsscript/termshell_gnu.py, mathicsscript/termshell_prompt.py:
 	Handle keybindings in GNU Readline
 
 2022-02-15  R. Bernstein <rocky@users.noreply.github.com>
@@ -512,7 +512,7 @@
 
 2022-02-06  rocky <rb@dustyfeet.com>
 
-	* mathicsscript/completion.py, mathicsscript/termshell_gnu.py: 
+	* mathicsscript/completion.py, mathicsscript/termshell_gnu.py:
 	Imports in core have changed Specifically where strip_context is located
 
 2021-12-19  rocky <rb@dustyfeet.com>
@@ -538,7 +538,7 @@
 2021-09-04  rocky <rb@dustyfeet.com>
 
 	* mathicsscript/data/inputrc-no-unicode,
-	mathicsscript/data/inputrc-unicode, mathicsscript/termshell_gnu.py: 
+	mathicsscript/data/inputrc-unicode, mathicsscript/termshell_gnu.py:
 	GNU readline was reading inputrc from wrong place Update readline tables.  Warning: input-no-unicode is probably wrong
 
 2021-09-04  rocky <rb@dustyfeet.com>
@@ -700,7 +700,7 @@
 
 2021-06-30  rocky <rb@dustyfeet.com>
 
-	* .pre-commit-config.yaml, mathicsscript/asymptote.py, setup.py: 
+	* .pre-commit-config.yaml, mathicsscript/asymptote.py, setup.py:
 	Start using Asymptote
 
 2021-06-30  rocky <rb@dustyfeet.com>
@@ -785,7 +785,7 @@
 
 2021-06-15  rocky <rb@dustyfeet.com>
 
-	* mathicsscript/bindkeys.py, requirements-extra.txt, setup.py: 
+	* mathicsscript/bindkeys.py, requirements-extra.txt, setup.py:
 	PyQT5: extra module for matplotlib
 
 2021-06-15  rocky <rb@dustyfeet.com>
@@ -906,7 +906,7 @@
 
 2021-05-27  rocky <rb@dustyfeet.com>
 
-	* mathicsscript/mmalexer.py, mathicsscript/termshell.py, setup.py: 
+	* mathicsscript/mmalexer.py, mathicsscript/termshell.py, setup.py:
 	WIP start using prompt-toolkit
 
 2021-05-15  R. Bernstein <rocky@users.noreply.github.com>
@@ -1244,7 +1244,7 @@
 
 2021-01-09  rocky <rb@dustyfeet.com>
 
-	* NEWS.md, mathicsscript/__main__.py, mathicsscript/termshell.py: 
+	* NEWS.md, mathicsscript/__main__.py, mathicsscript/termshell.py:
 	`-e` `--execute` is better suited for embedded use This options adds `--quiet` and shows just evaluation output as
 	text--execute
 
@@ -1273,7 +1273,7 @@
 
 2021-01-03  rocky <rb@dustyfeet.com>
 
-	* mathicsscript/termshell.py: Guard agains None eval returns
+	* mathicsscript/termshell.py: Guard against "None" eval returns
 
 2021-01-02  rocky <rb@dustyfeet.com>
 
@@ -1356,7 +1356,7 @@
 
 2020-12-22  R. Bernstein <rocky@users.noreply.github.com>
 
-	* : Merge pull request #7 from Mathics3/unicode-option Start to suport Unicode as a CLI option
+	* : Merge pull request #7 from Mathics3/unicode-option Start to support Unicode as a CLI option
 
 2020-12-22  rocky <rb@dustyfeet.com>
 
@@ -1373,13 +1373,13 @@
 	* mathicsscript/__main__.py, mathicsscript/format.py,
 	mathicsscript/inputrc, mathicsscript/inputrc-no-unicode,
 	mathicsscript/inputrc-unicode, mathicsscript/termshell.py,
-	setup.py, test/test_completion.py: Start to suport Unicode as a CLI
+	setup.py, test/test_completion.py: Start to support Unicode as a CLI
 	option
 
 2020-12-12  rocky <rb@dustyfeet.com>
 
 	* mathicsscript/__main__.py, mathicsscript/termshell.py: Handle
-	Unicode/WL charcter code mismatches
+	Unicode/WL character code mismatches
 
 2020-12-12  rocky <rb@dustyfeet.com>
 
@@ -1397,7 +1397,7 @@
 
 2020-12-08  rocky <rb@dustyfeet.com>
 
-	* mathicsscript/format.py: Use equal aspect ratio on ciruclar graphs
+	* mathicsscript/format.py: Use equal aspect ratio on circular graphs
 
 2020-12-07  rocky <rb@dustyfeet.com>
 
@@ -1405,7 +1405,7 @@
 
 2020-12-07  rocky <rb@dustyfeet.com>
 
-	* mathicsscript/__main__.py, mathicsscript/format.py: 
+	* mathicsscript/__main__.py, mathicsscript/format.py:
 	FindSpannngTree: use layout style of parent graph
 
 2020-12-07  rocky <rb@dustyfeet.com>
@@ -1439,7 +1439,7 @@
 
 2020-12-04  rocky <rb@dustyfeet.com>
 
-	* mathicsscript/format.py: Handel labeled nodes better Change color to white if we have node labels Change code slightly to facilitate expanding.
+	* mathicsscript/format.py: Handle labeled nodes better Change color to white if we have node labels Change code slightly to facilitate expanding.
 
 2020-12-04  rocky <rb@dustyfeet.com>
 
@@ -1511,7 +1511,7 @@
 
 2020-11-27  rocky <rb@dustyfeet.com>
 
-	* mathicsscript/mmalexer.py, mathicsscript/termshell.py, setup.py: 
+	* mathicsscript/mmalexer.py, mathicsscript/termshell.py, setup.py:
 	Can't use mathematica.lexer for now..  Sigh. It has promise. But recent Python gives now gives these...      FutureWarning: Possible nested set at position ...
 
 2020-11-27  rocky <rb@dustyfeet.com>
@@ -1594,7 +1594,7 @@
 2020-11-18  rocky <rb@dustyfeet.com>
 
 	* mathicsscript/__main__.py, mathicsscript/settings.m,
-	mathicsscript/termshell.py, settings/settings.m, setup.py: 
+	mathicsscript/termshell.py, settings/settings.m, setup.py:
 	Miscellaneous changes Settings`$ShowFullForm -> Settings`$ShowFullFormInput The old name has already caused too much confusion.  Check Pygments style and give a columnized list when the name is
 	invalid. Also make sure to reset the Mathics variable to a the last
 	valid value when an invalid one is set.
@@ -1686,7 +1686,7 @@
 
 	* NEWS.md, admin-tools/git2cl, admin-tools/make-dist.sh,
 	admin-tools/pyenv-versions, mathicsscript/term_background.py,
-	mathicsscript/version.py, setup.py, test/test_term_background.py: 
+	mathicsscript/version.py, setup.py, test/test_term_background.py:
 	Get ready for release 1.1.0rc1
 
 2020-10-21  R. Bernstein <rocky@users.noreply.github.com>
@@ -1777,4 +1777,3 @@
 
 	* Cull code from Mathics/main and revise ...  colorama is no longer optional.  We'll be using click soon.  Start
 	to pull in my better terminal-detection script.
-
