--- ChangeLog	2024-08-09 19:09:43.824371249 -0400
+++ ChangeLog-spell-corrected	2024-08-09 19:08:55.526614192 -0400
@@ -53,7 +53,7 @@
 2023-02-25  rocky <rb@dustyfeet.com>

 	* mathicsscript/__main__.py, mathicsscript/asymptote.py,
-	mathicsscript/fake_psviewer.py, mathicsscript/format.py: Removce
+	mathicsscript/fake_psviewer.py, mathicsscript/format.py: Remove
 	flake8 things

 2023-02-25  rocky <rb@dustyfeet.com>
@@ -1088,12 +1088,12 @@
 	* mathicsscript/__main__.py, mathicsscript/format.py,
 	mathicsscript/inputrc, mathicsscript/inputrc-no-unicode,
 	mathicsscript/inputrc-unicode, mathicsscript/termshell.py,
-	setup.py, test/test_completion.py: Start to suport Unicode as a CLI
+	setup.py, test/test_completion.py: Start to support Unicode as a CLI
 	option

 2020-12-12  rocky <rb@dustyfeet.com>

 	* mathicsscript/__main__.py, mathicsscript/termshell.py: Handle
-	Unicode/WL charcter code mismatches
+	Unicode/WL character code mismatches

 2020-12-12  rocky <rb@dustyfeet.com>

@@ -1154,6 +1154,6 @@

 2020-12-04  rocky <rb@dustyfeet.com>

-	* mathicsscript/format.py: Handel labeled nodes better Change color to white if we have node labels Change code slightly to facilitate expanding.
+	* mathicsscript/format.py: Handle labeled nodes better Change color to white if we have node labels Change code slightly to facilitate expanding.

 2020-12-04  rocky <rb@dustyfeet.com>
